packages/web-client/app/templates/card-pay/payments.hbs

```diff
@@ -51,19 +55,21 @@
       <header class="card-pay-dashboard__payments-section-header">
         <h3 class="card-pay-dashboard__payments-title">Payment Profiles</h3>
         {{#let @model.panel.sections.[0] as |section|}}
-          <Boxel::Button
-            {{on "click" (fn this.transitionToWorkflow section.workflow)}}
-            @kind="secondary-dark"
-            @size="small"
-            @disabled={{section.isCtaDisabled}}
-            data-test-workflow-button={{section.workflow}}
-          >
-            <span>{{section.cta}}</span>
-            {{svg-jar "plus"
-              class="card-pay-dashboard__payments-section-header-button-icon"
-              role="presentation"
-            }}
-          </Boxel::Button>
+          {{#unless this.merchantSafes.length}}
```

Comment:
maybe this reaches the threshold where it would make sense to extract? Ideally it would be near-global to the “account” somehow but even just a getter is fine probs

Reply:
`safes.hasMerchant` might work